beacon_node/beacon_chain/src/schema_change.rs

```diff
@@ -73,6 +75,28 @@ pub fn migrate_schema<T: BeaconChainTypes>(
 
             Ok(())
         }
+        // Migration for adding `is_merge_complete` field to the fork choice store.
+        (SchemaVersion(4), SchemaVersion(5)) => {
```

Comment:
There's a schema 5 in `unstable` already for checkpoint sync, so this should be 5->6 once `merge-f2f` gets updated for the latest `unstable`